CONTRIBUTING.md

```diff
@@ -354,6 +354,22 @@ You can rebuild iD completely with the command `npm run all`.
 iD will be built to the `dist` directory. This directory is self-contained; you can copy it
 into the public directory of your webserver to deploy iD.
 
+### Build Troubleshooting
+
+**`npm install` fails with "PhantomJS not found on PATH … Try to install
+PhantomJS globally".**
+Then you should do exactly that, such as `apt install phantomjs`.
+
+**`npm install` fails with "Command failed: /usr/bin/phantomjs --version
+qt.qpa.screen: QXcbConnection: Could not connect to display".**
```

Comment:
What OS did you encounter this on? This looks like a bad phantomjs install, not an iD issue